pluto_build/bin/config.sh

```diff
@@ -31,34 +31,22 @@ function urlparse {
 }
 
 function FGDB_export {
+  name=$1
   urlparse $BUILD_ENGINE
-  mkdir -p output/$@.gdb &&
-  (cd output/$@.gdb
-    docker run \
-      --network host\
-      -v $(pwd):/data\
-      --user $UID\
-      --rm webmapp/gdal-docker:latest ogr2ogr -progress -f "FileGDB" $@.gdb \
-        PG:"host=$BUILD_HOST user=$BUILD_USER port=$BUILD_PORT dbname=$BUILD_DB password=$BUILD_PWD" \
-        -mapFieldType Integer64=Real\
-        -lco GEOMETRY_NAME=Shape\
-        -nln $@\
-        -nlt MULTIPOLYGON\
-        $@
-    docker run \
-      --network host\
-      -v $(pwd):/data\
-      --user $UID\
-      --rm webmapp/gdal-docker:latest ogr2ogr -progress -f "FileGDB" $@.gdb \
-        PG:"host=$BUILD_HOST user=$BUILD_USER port=$BUILD_PORT dbname=$BUILD_DB password=$BUILD_PWD" \
-        -mapFieldType Integer64=Real\
-        -update -nlt NONE\
-        -nln NOT_MAPPED_LOTS\
-        unmapped
-      rm -f $@.gdb.zip
-      zip -r $@.gdb.zip $@.gdb
-      rm -rf $@.gdb
-    )
+  mkdir -p output/$name &&
+    (cd output/$name
+      ogr2ogr -progress -f "FileGDB" $name.gdb \
+          PG:"host=$BUILD_HOST user=$BUILD_USER port=$BUILD_PORT dbname=$BUILD_DB password=$BUILD_PWD" \
+          -mapFieldType Integer64=Real\
+          -lco GEOMETRY_NAME=Shape\
+          -nln $name\
+          -nlt MULTIPOLYGON\
+          $name 
+      rm -f $name.gdb.zip
```

Comment:
This doesn't replicate the exact same removes from the previous functionality - may need to be updated to reflect